readme.md

```diff
@@ -2,10 +2,6 @@
 
 -----------------
 
```

Comment:
Shouldn't it be left in place?

Reply:
@dorodnic Can you comment? Should this stay in, or be removed?

Reply:
I don't see the value. When it was multiple systems it made sense. Today its just Travis, and we check before merging
If Gated passed but Push build failed, what would you do with it?